src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateRes = date.split(fromFormat[3]);
+  const result = [];
+  let year = 0;
+  let month = 0;
+  let day = 0;
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'YY':
+        year = i;
+        break;
+      case 'YYYY':
+        year = i;
+        break;
+      case 'MM':
+        month = i;
+        break;
+      case 'DD':
+        day = i;
+        break;
+    }
```

Comment:
Good practice to add spaces between cases
```suggestion
    switch (fromFormat[i]) {
      case 'YY':
        year = i;
        break;
      
      case 'YYYY':
        year = i;
        break;
       
      case 'MM':
        month = i;
        break;
        
      case 'DD':
        day = i;
        break;
    }
```